DecryptTransformer.js

```diff
@@ -26,6 +26,12 @@ export class DecryptTransformer {
     keyAgreement,
     keyAgreementKey
   } = {}) {
+    if(!keyAgreement) {
+      throw new SyntaxError('"keyAgreement" is a required parameter.');
+    }
+    if(!keyAgreementKey) {
+      throw new SyntaxError('"keyAgreementKey" is a required parameter.');
```

Comment:
```suggestion
      throw new TypeError('"keyAgreementKey" is a required parameter.');
```

Reply:
I referred to this to select a `SyntaxError` for this purpose: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Missing_formal_parameter

Reply:
That is not referring to missing parameters when a function is called, but rather a syntax error when declaring a function (i.e., it won't even parse as valid JS):

```js
function square(3) {
  return number * number;
};
// SyntaxError: missing formal parameter

function greet("Howdy") {
  return greeting;
};
// SyntaxError: missing formal parameter

function log({ obj: "value"}) { 
  console.log(arg)
};
// SyntaxError: missing formal parameter
```

Reply:
See this instead: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/More_arguments_needed